cypress/integration/rendering/gantt.spec.js

```diff
@@ -523,6 +523,108 @@ describe('Gantt diagram', () => {
     );
   });
 
+  it('should render when there is a dateRange', () => {
+    imgSnapshotTest(
+      `
+      ---
+      displayMode: compact
+      ---
+      gantt
+        title GANTT compact
+        dateFormat  HH:mm:ss
+        dateRange   12:30:00,13:30:00
+        axisFormat  %Hh%M
+        section DB Clean
```

Comment:
Please add more test cases that handle dates as well, currently only time is used. 
Then some cases where there are tasks before and after the given range.

Reply:
I've started, but am new to cypress and having difficulties getting the tests to pass locally / produce screenshots that look right. Will keep going at it over the next few days and post in the Discord if I have any specific questions!